src/convertToObject.js

```diff
@@ -6,7 +6,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const input = sourceString.split(';');
+  const result = input.reduce((acc, property) => {
```

Comment:
res/result/el/elem/arr/array/obj/object and etc are bad names, variable name should be descriptive